### ISSUE TITLE
Importing mui-nested-menu forces the bundling of the entire MUI package

### ISSUE THREAD

**Issue by jraoult** [steviebaa/mui-nested-menu#17]
After recently adding `mui-nested-menu` in my app, I noticed a big jump in the output bundle generated by webpack. After digging into the published code, I concluded that it is impossible for webpack (and probably other bundlers) to three shakes the dependencies.

The "culprit" is, for example on [line 23 of this file  ](https://unpkg.com/browse/mui-nested-menu@2.1.0/components/ContextMenu.js):

```
var material_1 = require("@mui/material");
```

When a bundle sees that, it can not know exactly which module is actually used and will import the whole package.

There are different ways to fix this issue, but I would recommend the [option 1 recommended by the MUI team](https://mui.com/material-ui/guides/minimizing-bundle-size/#option-1.).

So, for example, instead of:

```
import {Menu, MenuItemProps, MenuProps} from '@mui/material';
```

doing

```
import Menu, {MenuItemProps, MenuProps} from '@mui/material/Menu';
```

**Comment by steviebaa** [comment]
Great spot @jraoult! The example looks like a clean fix. Would you like to make a PR? Otherwise I'll try to find some time over the weekend.

**Comment by steviebaa** [comment]
Published to npm as [2.1.1](https://www.npmjs.com/package/mui-nested-menu/v/2.1.1).

See #18 for details.
